### PR TITLE
Add gatsby starter list.json

### PR DESCRIPTION
**Pull request by Simply007**
### Motivation

New template submission.

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

No really.

### CODE REVIEW

File: templates/list.json [hunk 1]
```diff
@@ -96,5 +96,19 @@
...
+        "exportPackageUrl": "https://github.com/Kentico/gatsby-starter-kontent/archive/master.zip",
+        "imageUrl": "https://raw.githubusercontent.com/Kentico/gatsby-starter-kontent/master/template.jpg",
+        "author": {
+            "name": "Ondřej Chrastina",
+            "email": "ondrej.chrastina@kentico.com"
+        },
+        "lastUpdate": "2020-05-13T09:13:05.794Z",
+        "forVersion": "1",
+        "technology": "GatsbyJs"
+    },
```
Comment (Simply007) [inline templates/list.json:113]:
```suggestion
    }
```

**Comment by Simply007** [comment]
Unfortunately, template manager is loading JSON directly from master, so Netlify preview is not able to showcase the preview with newly added item.

I would vote to use local file (include the JSON in the build - not sure how - and then load it as a local source using HTTP client `/templates/list.json`).

What do you think @Enngage?

**Comment by Enngage** [comment]
@Simply007, I could surely do that, but from what location would I load the file?

**Comment by Enngage** [comment]
Could you possible create an issue, please? :)

**Comment by Simply007** [comment]
There you go @Enngage 
https://github.com/Kentico/kontent-template-manager/issues/40

**Pull request merged**